graphs/minimum_spanning_tree_prims2.py

```diff
@@ -0,0 +1,262 @@
+from sys import maxsize
```

Comment:
@ruppysuppy this repository is for educational purpose, so kindly add a description of your work